src/Model/ModelManager.php

```diff
@@ -445,7 +456,7 @@ public function modelReverseTransform($class, array $array = [])
 
             if ($reflClass->hasMethod($setter)) {
                 if (!$reflClass->getMethod($setter)->isPublic()) {
-                    throw new PropertyAccessDeniedException(sprintf('Method "%s()" is not public in class "%s"', $setter, $reflClass->getName()));
+                    throw new \BadMethodCallException(sprintf('Method "%s()" is not public in class "%s"', $setter, $reflClass->getName()));
```

Comment:
This is a BC-break, isn't it ?

Reply:
I have to check again what I did, probably split the two commits in two PR.

Reply:
[The class `PropertyAccessDeniedException` was remoted in Symfony 2.3](https://github.com/symfony/symfony/blob/dd4d393a3c9632fdecd14fb1c1f94e665f588451/src/Symfony/Component/PropertyAccess/CHANGELOG.md#230) 🤣